store/src/test/java/org/apache/rocketmq/store/DefaultMessageStoreTest.java

```diff
@@ -148,8 +149,10 @@ public void testPullSize() throws Exception {
             messageExtBrokerInner.setQueueId(0);
             messageStore.putMessage(messageExtBrokerInner);
         }
-        //wait for consume queue build
-        Thread.sleep(10);
+        // wait for consume queue build
+        // the sleep time should be great than
+        // commit log flush interval plus consume queue flush interval
```

Comment:
Such comment is not true, for the disk flush type is SYNC_FLUSH. It only need time for build consume queue.
Normally 10 ms is enough, but for safe, waiting 100ms is OK.

Reply:
Yes, it only needs time for building consume queue, not include commit log flush interval